sidekiq-debounce.gemspec

```diff
@@ -25,10 +25,11 @@ DESC
   spec.test_files    = spec.files.grep(%r{^spec/})
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'sidekiq', '>= 3.1.0'
+  spec.add_dependency 'sidekiq', '>= 2.17'
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'bundler', '~> 1.6'
   spec.add_development_dependency 'mock_redis'
   spec.add_development_dependency 'mocha'
   spec.add_development_dependency 'codeclimate-test-reporter'
+  spec.add_development_dependency 'minitest'
```

Comment:
The commit message says there was a problem requiring `minitest/spec`, but that should ship with modern rubies.  What's your Ruby flavor+version?  We might wanna add it to the build matrix